packages/css/src/components/blockquote/blockquote.scss

```diff
@@ -21,7 +21,7 @@
   line-height: var(--ams-blockquote-line-height);
   quotes: "“" "”";
 
-  @include hyphenation;
+  @include hyphenation.hyphenation;
   @include text-rendering;
```

Comment:
Mixin two approaches isn’t ideal.

Reply:
I agree. It might be better to bundle our mixins like so:

```css
@include text.hyphenation;
@include text.rendering;
```